lib/OpenQA/CacheService/Task/Asset.pm

```diff
@@ -33,13 +33,14 @@ sub _cache_asset {
     return $job->finish unless defined $asset_name && defined $type && defined $host && defined $lock;
     my $guard = $app->progress->guard($lock);
     unless ($guard) {
-        $job->note(output => 'Asset was already requested by another job');
+        $job->note(
+            output => "Asset $asset_name was downloaded by another job, details are therefore unavailable to us");
```

Comment:
the "to us" is not making it clearer what this refers to. I suggest to simply use "here"